lib/class-wp-rest-swagger-controller.php

```diff
@@ -350,11 +352,8 @@ private function schemaIntoDefinition($schema){
 //--
 
 			
-			if($prop['type']=='array'){
-				$prop['items'] = array(
-					'type'=> $prop['items']['type'],
-					'properties'=> $prop['items']['properties']
-				);
+			if($prop['type']=='array' && !$prop['items']){
+				$prop['items']=array('type'=>'string');
```

Comment:
I noticed I was effectively assigning this property to itself...the property items are already set, they were just overriding themselves to string. This works as a fallback in this manner I believe.